dev/cognitiveservices/data-plane/Language/common.json

```diff
@@ -1540,12 +1538,11 @@
   },
   "parameters": {
     "Endpoint": {
-      "name": "Endpoint",
+      "name": "endpoint",
```

Comment:
In /specification folder, the latest stable is uppercase "Endpoint", and latest preview is lowercase "endpoint", so no matter what we do here, it is a breaking change. So I will revert this to keep it in uppercase "Endpoint" so no breaking change to GA version. @heaths, Is that OK?